test/e2e/azure_machinepool_drain.go

```diff
@@ -62,6 +63,7 @@ type (
 		ClusterName           string
 		SkipCleanup           bool
 		IPv6                  bool
+		IPFamilies            []corev1.IPFamily
```

Comment:
do we need both IPv6 and IPFamilies here? Seems like they are both conveying the same info, could we get rid of the IPv6 bool and deduct it from the IPFamilies input?

Reply:
If these tests are for 1.20+, I think we can rely on `IPFamilies[0]`. I've pushed an update to test it.